integration/spark/src/test/scala/org/apache/carbondata/view/timeseries/TestMVTimeSeriesLoadAndQuery.scala

```diff
@@ -373,8 +363,8 @@ class TestMVTimeSeriesLoadAndQuery extends QueryTest with BeforeAndAfterAll {
     sql("drop table if exists maintable")
   }
 
-  def dropMaterializedView(mvName: String): Unit = {
```

Comment:
why it is required? both are same

Reply:
Sorry, I cann't get the point.

Reply:
you have changed method name from `dropMaterializedView` to `dropMV`. Can keep the old method name.

Reply:
Oh, i want to use abbreviation here like other mv method.